benchmark/schema/preconditioner.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Preconditioner",
+  "description": "Input for a single preconditioner operation",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "spmv.reuse.json"
+    },
+    {
+      "properties": {
+        "operator": {
+          "$ref": "operator.json"
```

```suggestion
          "$ref": "operator.reuse.json"
```
follow spmv.reuse.json?